projects/blueberry/index.js

```diff
@@ -8,7 +8,7 @@ const sdk = require('@defillama/sdk');
 const BANK = '0xa34F59F634d48E2c3606048f2367326c46a4B5fA';
 
 async function tvl(timestamp, block, chainBlocks, {api}) {
-  const positionValues = await api.fetchList({  lengthAbi: abi.getNextPositionId, itemAbi: abi.getPositionValue, target: BANK})
```

Comment:
similar check for numm/undefined here?

Reply:
Similar fix to the previous PR, added a check in case the multicall fails or returns null/undefined
```
  const positionValuesRes = await api.fetchList({  lengthAbi: abi.getNextPositionId, itemAbi: abi.getPositionValue, target: BANK, permitFailure: true})
  const positionValues = positionValuesRes.filter(value => value != null)
```